wayland-cursor/src/lib.rs

```diff
@@ -160,17 +161,18 @@ impl CursorTheme {
             pool,
             pool_size: INITIAL_POOL_SIZE,
             cursors: Vec::new(),
+            backend: conn.backend().weak(),
```

Comment:
To match Rc and Arc, shouldn't that function be called `downgrade` instead?

Reply:
I guess :shrug: